### PR TITLE
Checkout: add support for Giropay and Bancontact

### PR DESCRIPTION
**Pull request by yoavf**
See internal D8081-code  
This builds on #17754, so the additional changes are rather small.

screen shot 2017-11-02 at 17 29 56

To test, make sure you set your abtest value to show (and use a Germany/Belgium IP address):
`localStorage.setItem( 'ABTests', '{"showNewPaymentMethods":"show"}' )`

Depends on #19696

### CODE REVIEW

**Comment by yoavf** [comment]
Tests are likely to fail on default payment methods :)

**Comment by yoavf** [comment]
Lots of tests fails because of the abtest in `getCurrentUserPaymentMethods` - I'll see about moving the abtest to the payment box class

File: client/lib/cart-values/index.js [hunk 1]
```diff
@@ -170,6 +174,23 @@ function isNetherlandsIdealEnabled( cart ) {
...
+	return (
+		config.isEnabled( 'upgrades/germany-giropay' ) &&
+		cart.allowed_payment_methods.indexOf( 'WPCOM_Billing_Stripe_Source_Giropay' ) >= 0 &&
+		'EUR' === cart.currency
+	);
+}
+
+function isBelgiumBancontactEnabled( cart ) {
+	return (
+		true ||
```
Comment (akirk) [inline client/lib/cart-values/index.js]: Left-over debug code

File: client/lib/abtest/active-tests.js [hunk 2]
```diff
@@ -117,4 +117,14 @@ export default {
 		allowExistingUsers: true,
 		localeTargets: 'any',
 	},
+	showNewPaymentMethods: {
+		datestamp: '20171113',
```
Comment (akirk) [inline client/lib/abtest/active-tests.js]: Update the timestamp?

**Pull request merged**